app/views/schools/dashboard/index.html.erb

```diff
@@ -22,4 +22,4 @@
     <% end %>
   </tbody>
 </table>
-<%= govuk_paginate @schools %>
+<%= govuk_paginate @schools, @pagy %>
```

Comment:
one of you other changes made it `<%==`. do you need the double equals here? or do you not need the double equals in the other place?

Reply:
yes it is needed 👍
nice catch